rails-tasks/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>note to self<br>must do:</h1>
+<table align="center">
+  <% @tasks.each do |task| %>
+    <tr>
+      <td>
+        <% if task.complete == true %>
```

Comment:
Since you're checking against `true` this condition can simply say `if task.complete`